clang/include/clang/Sema/Sema.h

```diff
@@ -12836,6 +12834,31 @@ class Sema final {
   }
 };
 
+template <typename AttrType>
+void Sema::addIntelSYCLFunctionAttr(Decl *D, const AttributeCommonInfo &Attr,
```

Comment:
```suggestion
void Sema::addIntelSYCLSingleArgFunctionAttr(Decl *D, const AttributeCommonInfo &Attr,
```

There are more (triple arguments) function attributes, so I suggest this change.

Reply:
Done